statistics/handle/handle_test.go

```diff
@@ -950,6 +950,7 @@ func (s *testSerialStatsSuite) checkForGlobalStatsWithOpts(c *C, tk *testkit.Tes
 }
 
 func (s *testSerialStatsSuite) TestAnalyzeGlobalStatsWithOpts1(c *C) {
+	c.Skip("unstable test")
```

Comment:
This test is itself unstable, you can try this many times, and may encounter failure:

```
make ut X='run statistics/handle testSerialStatsSuite.TestAnalyzeGlobalStatsWithOpts1'
```

Reply:
https://github.com/pingcap/tidb/issues/32035